### PR TITLE
Eslint minify

### PR DESCRIPTION
**Pull request by JanSunavec**
Eslint does not minify JS and also append @lightningjs/core package into appBundle.js file. Result is appBundle file with enormous  file size. 

This PR, minify source code and remove litghtning.js dependency. Output is less JS code in appBundle. Much better result than rollup.

### CODE REVIEW

**Comment by JanSunavec** [comment]
@michielvandergeest Yet another PR. :-)

File: src/configs/esbuild.es5.config.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ module.exports = (folder, globalName) => {
       alias([
         { find: '@', filter: /@\//, replace: path.resolve(process.cwd(), 'src/') },
         { find: '~', filter: /~\//, replace: path.resolve(process.cwd(), 'node_modules/') },
+        { find: '@lightningjs/core', filter: /^@lightningjs\/core$/, replace: path.join(__dirname, '../alias/lightningjs-core.js'),},
```
Comment (michielvandergeest) [inline src/configs/esbuild.es5.config.js:54]: Thanks for finding this one and bringing it up to par with the rollup version!

File: src/configs/esbuild.es5.config.js [hunk 2]
```diff
@@ -81,12 +82,14 @@ module.exports = (folder, globalName) => {
         },
       }),
     ],
+    minifyWhitespace: true,
+    minifyIdentifiers: true,
+    minifySyntax: false,
```
Comment (michielvandergeest) [inline src/configs/esbuild.es5.config.js:87]: Can we make minification configurable similar to the rollup version?
https://github.com/rdkcentral/Lightning-CLI/blob/62bd1234f57d3be7b2442faf58d95e57d10a9d50/src/configs/rollup.es6.config.js#L63-L64

**Review by michielvandergeest — CHANGES_REQUESTED** [review]

File: src/configs/esbuild.es5.config.js (hunk 2 above)
Comment (JanSunavec) [reply src/configs/esbuild.es5.config.js:88]: Sure. Please check now..

File: src/configs/esbuild.es5.config.js (hunk 1 above)
Comment (JanSunavec) [reply src/configs/esbuild.es5.config.js:55]: You are welcome.

**Review by michielvandergeest — APPROVED** [review]
LGTM!

**Pull request merged**